content/en/docs/ops/diagnostic-tools/multicluster/index.md

```diff
@@ -121,8 +121,8 @@ To verify certs are configured correctly, you can compare the root-cert in each
 
 {{< text bash >}}
 $ diff \
-   <(kubectl --context="${CTX_CLUSTER1}" -n istio-system get secret cacerts -ojsonpath='{.data.root-cert\.pem}') \
-   <(kubectl --context="${CTX_CLUSTER2}" -n istio-system get secret cacerts -ojsonpath='{.data.root-cert\.pem}')
+   <(kubectl --context="${CTX_CLUSTER1}" -n istio-system get secret istio-ca-secret -ojsonpath='{.data.ca-cert\.pem}') \
+   <(kubectl --context="${CTX_CLUSTER2}" -n istio-system get secret istio-ca-secret -ojsonpath='{.data.ca-cert\.pem}')
```

Comment:
This change does not look right. The current names (`cacerts` and `root-cert.pem`) are exactly the ones used in the referenced document used for setting up certs for the multicluster installation docs. https://istio.io/latest/docs/tasks/security/cert-management/plugin-ca-cert/#plug-in-certificates-and-key-into-the-cluster

Reply:
I'm going through this at the moment to get multi cluster routing working. But there were no such secrets in my istio deployments and no such keys. I'm working with version `19.1`. 
I was using helm to install.
If it happens these secrets appear, I'll update this PR with note about them not being present.

Reply:
The multicluster installation before-you-begin section creates the secret: https://istio.io/latest/docs/setup/install/multicluster/before-you-begin/#configure-trust

There is a tip that you can avoid this if you're just deploying a single control plane (simple primary-remote setup), but none of this applies in that case anyway. This is dealing with the general case of multiple primary clusters where a shared root of trust is needed.